### PR TITLE
Feature/implement scraper

### PR DESCRIPTION
**Pull request by KarmaKamikaze**

### CODE REVIEW

**Review by selectionator — APPROVED** [review]
LGTM 👍
uwu

**Review by Sarmisuper — APPROVED** [review]
Brutal but efficient

File: HotelPriceScout/Data/Interface/SettingsManager.cs [hunk 1]
```diff
@@ -22,6 +23,30 @@ public class SettingsManager : ComponentBase
...
+                    break;
+                case 2:
+                    result.Add(timeValuePass);
+                    result.Add(timeValuePass2);
+                    break;
+                case 3:
+                    result.Add(timeValuePass);
+                    result.Add(timeValuePass2);
+                    result.Add(timeValuePass3);
+                    break;
```
Comment (Sarmisuper) [inline HotelPriceScout/Data/Interface/SettingsManager.cs]: use fall-through here instead by switching case numbers and then removing breaks.

Comment (Sarmisuper) [reply HotelPriceScout/Data/Interface/SettingsManager.cs]: of course husk en break inden default

**Review by Sarmisuper — APPROVED** [review]
ye

**Review by Sarmisuper — APPROVED** [review]
LGTT

**Review by selectionator — APPROVED** [review]
LGTM

**Pull request closed without merging**